react/README.md

```diff
@@ -0,0 +1,59 @@
+<p align="center">
+  <a href="https://heroicons.com" target="_blank">
+    <picture>
+      <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/tailwindlabs/heroicons/HEAD/.github/logo-dark.svg">
+      <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/tailwindlabs/heroicons/HEAD/.github/logo-light.svg">
+      <img alt="Heroicons" width="315" height="117" style="max-width: 100%" src="https://raw.githubusercontent.com/tailwindlabs/heroicons/HEAD/.github/logo-light.svg">
+    </picture>
+  </a>
+</p>
+
+<p align="center">Beautiful hand-crafted SVG icons, by the makers of Tailwind CSS.<p>
+
+<p align="center">
+  <a href="https://heroicons.com"><strong>Browse at Heroicons.com &rarr;</strong></a>
+</p>
+
+<p align="center">
+    <a href="https://github.com/tailwindlabs/heroicons/releases"><img src="https://img.shields.io/npm/v/heroicons" alt="Latest Release"></a>
+    <a href="https://github.com/tailwindlabs/heroicons/blob/master/LICENSE"><img src="https://img.shields.io/npm/l/heroicons.svg" alt="License"></a>
```

Comment:
these should point to the `@heroicons/react` package

Reply:
Good catch! 🤙